lib/served/backends.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'backends/base'
+module Served
+  module Backends
+
+    # @private
+    def self.[](backend)
+      @backends ||= {
+        http:     'HTTP',
+        patron:   'Patron',
+        httparty: 'HTTParty'
+      }
+      if @backends[backend]
+        require_relative "backends/#{backend}"
```

Comment:
You don't _have_ to use `require_relative` here since the gem is already presumably on the load path.  You could do `require "served/backends/#{backend}"`